radio/src/gui/128x64/view_statistics.cpp

```diff
@@ -117,6 +117,7 @@ void menuStatisticsView(event_t event)
   #define MENU_DEBUG_COL2_OFS          (17*FW)
   #define MENU_DEBUG_Y_CURRENT         (1*FH)
   #define MENU_DEBUG_ROW1              (1*FH+1)
+  #define MENU_DEBUG_ROW2              (2*FH+1)
   #define MENU_DEBUG_Y_MAH             (2*FH)
   #define MENU_DEBUG_Y_CPU_TEMP        (3*FH)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Remove the whitespaces before the # character. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=cpp%3APPDirectiveIndentation)